package.json

```diff
@@ -3408,7 +3408,7 @@
                                                         "name": "Timeline",
                                                         "icon": "clock",
                                                         "type": "command",
-                                                        "command": "flutter.openTimeline"
+                                                        "command": "files.openTimeline"
```

Comment:
Just double checking. Is the `files.openTimeline` correct replacement for this? This doesn't seem like a flutter specific command?

Reply:
You're absolutely correct, I thought it was another one of the dart commands that had been brought into native vscode, but it turns out that feature was just dropped from the dart extension because the information it provided was included elsewhere.  Thanks for catching that, change pushed!